include/inviwo/core/properties/ordinalproperty.h

```diff
@@ -224,6 +224,54 @@ IVW_CORE_API OrdinalPropertyState<vec3> ordinalLight(
     const vec3& pos, float min = -100.0, float max = 100.0,
     InvalidationLevel invalidationLevel = InvalidationLevel::InvalidOutput);
 
+template <typename T, typename U>
+OrdinalPropertyState<T> ordinalSymmetricVector(
```

Comment:
My thinking was that the symmetric part was for the fact the the min == -max
Having T and U being independent make it a bit easier since it will allow some more conversions...

Reply:
> Having T and U being independent make it a bit easier since it will allow some more conversions...

Are you sure? The OrdinalProperty does not differentiate between the types. 
Perhaps there should also be an origin to do the symmetry arround, defaulting to 0

Reply:
Point of using these convenience function is that they should be fast and easy to use, i.e. not more options then needed... otherwise one can just use the regular constructor with all the settings... 

Yes, I got exactly that compiler error as I said above... point here is that the min/max argument is just a scalar not a vector... i.e. I assume the min.x == min.y == min.z == min... to make it as easy to use as possible...